src/patternfly/components/Table/table.scss

```diff
@@ -60,7 +60,7 @@
 
   // Truncate
   --pf-c-table--m-truncate--cell--MaxWidth: 1px;
-  --pf-c-table--m-truncate--cell--MinWidth: 9ch;
+  --pf-c-table--m-truncate--cell--MinWidth: calc(5ch + var(--pf-c-table--cell--PaddingRight) + var(--pf-c-table--cell--PaddingLeft));
```

Comment:
I wonder if we should have a var for `5ch` that serves as the theme var for that width, so that's what a user changes to set this. Then we use it in this `MinWidth` var and add the padding to it.

Reply:
I did something similar in pagination to create a `--whatever--chars` var that is the number of characters you want to show in the pagination text input, and the react component takes a prop for how many chars to show that modifies that var's value.

https://github.com/patternfly/patternfly/blob/ceb30f2f12240f73063013a9edca7c9b002642d5/src/patternfly/components/Pagination/pagination.scss#L32